### ISSUE TITLE
Consider firefox image updates for stable tracks

### ISSUE THREAD

**Issue by Intensity** [jessfraz/dockerfiles#577]
Now that Debian bullseye is out, I wanted to open up the option of using this stable foundation (instead of `sid`) as the basis for a firefox image. (Another alternative still would be `buster`).

Similarly, what about using `firefox-esr` even from the native Debian repositories?

I've found that sound support has been hard for me to get right if I attempt to bake my own image, but I'm really not sure why it's so difficult. Anyhow I'm glad it's working for this image.

I'm bringing up the `firefox-esr` and stable distribution version ideas as suggestions to increase the overall stability and longevity of an image. If all aspects, including sound, end up working fine in a more stable track, this could represent a viable alternative.

**Comment by ngaro** [comment]
I agree with most things you mention, it's all fixed in my PR #527 from 2 years ago. I'm still waiting until it gets merged... Meanwhile you can use the Dockerfile in my fork (make sure you use the correct branch).
